neuralprophet/conformal.py

```diff
@@ -190,3 +190,62 @@ def plot(self, plotting_backend: str):
             )
         if plotting_backend in ["matplotlib", "plotly"] and matplotlib.is_interactive():
             fig.show()
+
+    def evaluate(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Evaluate conformal prediction on test dataframe.
+
+        Parameters
+        ----------
+            df : pd.DataFrame
+                test dataframe with the conformal prediction intervals
+
+        Returns
+        -------
+            pd.DataFrame
+                table containing evaluation metrics such as interval_width and miscoverage_rate
```

Comment:
Does is not only return interval_width and miscoverage_rate? Maybe just specify: 
```suggestion
                table containing the two evaluation metrics interval_width and miscoverage_rate for each predicted interval
```